.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -24,12 +24,20 @@ body:
       description: Provide a short description of the problem.
     validations:
       required: true
+      
+  - type: textarea
+    id: os-version
+    attributes:
+      label: Operating System
+      description: What Operating System are you using? Linux, Mac OS, WSL on Windows 
+    validations:
+      required: true
```

Comment:
Can we make it non-required? Sometimes OS really doesn't matter. We already have a long list of mandatory fields that people have to fill in. Making the list longer will make people angrier 😆

Reply:
@li-boxuan removed the required haha